### PR TITLE
Add more and better logging to JHU indicator

### PR DESCRIPTION
**Pull request by benjaminysmith**
### Description
Add more and better logging to JHU indicator

### Changelog
- Return list of dates written by the CSV exporter
- Log the # of CSV exports, oldest final date written, lag, and execution time for JHU indicator.

### Fixes 
- Improves on #606

### CODE REVIEW

File: jhu/delphi_jhu/run.py [hunk 1]
```diff
@@ -4,8 +4,10 @@
 This module should contain a function called `run_module`, that is executed
 when the module is run with `python -m MODULE_NAME`.
 """
-from datetime import datetime
+from datetime import datetime as dt
 from itertools import product
+import datetime
```
Comment (sgsmob) [inline jhu/delphi_jhu/run.py:9]: I don't think this `import datetime` is needed.

Comment (sgsmob) [reply jhu/delphi_jhu/run.py:9]: So you can probably also delete the alias of `datetime.datetime` to `dt` if you would like.

**Review by sgsmob — APPROVED** [review]
LGTM modulo the quick comments.

File: jhu/delphi_jhu/run.py (hunk 1 above)
Comment (benjaminysmith) [reply jhu/delphi_jhu/run.py:9]: I'm still using it on line 156 (datetime.datetime.now())

Comment (sgsmob) [reply jhu/delphi_jhu/run.py:9]: right but you have already imported `datetime.datetime` as `dt`

Comment (benjaminysmith) [reply jhu/delphi_jhu/run.py:9]: Oh nvm, I see what you mean. Removed.

**Pull request merged**